libs/SalesforceAnalytics/SalesforceAnalytics/Supporting Files/SalesforceAnalytics-Prefix.pch

```diff
@@ -4,4 +4,5 @@
 
 #ifdef __OBJC__
     #import <Foundation/Foundation.h>
+    #import "SFSDKAnalyticsLogger.h"
```

Comment:
It only does public header files, but you are right that it would be great if it did also prefix headers.